src/java.base/share/classes/javax/crypto/ShortBufferException.java

```diff
@@ -42,7 +42,7 @@ public class ShortBufferException extends GeneralSecurityException {
     private static final long serialVersionUID = 8427718640832943747L;
 
     /**
-     * Constructs a ShortBufferException with no detail
+     * Constructs a <code>ShortBufferException</code> with no detail
```

Comment:
Use `{@code }` for all of the javadoc in this class?

Reply:
using `{@code }`